stack/maxima/sandbox.wxm

```diff
@@ -1,40 +1,43 @@
 /* [wxMaxima batch file version 1] [ DO NOT EDIT BY HAND! ]*/
-/* [ Created with wxMaxima version 13.04.2 ] */
-
+/* [ Created with wxMaxima version 18.02.0 ] */
 /* [wxMaxima: title   start ]
 STACK Sandbox
    [wxMaxima: title   end   ] */
 
+
 /* [wxMaxima: comment start ]
 This document loads the extra files needed for STACK.
 See https://github.com/maths/moodle-qtype_stack
 
-1. Download the files from github, either using git or as a zip file.
-2. Place the files somewhere they can be read, and edit the line velow to give the location.
```

Comment:
Is it ever necessary to change this? Does sandbox.wxm not automatically look for the files in its own directory?